Sources/Build/Command.link(ClangModule).swift

```diff
@@ -28,7 +28,7 @@ extension Command {
             let buildMeta = ClangModuleBuildMetadata(module: module, prefix: prefix, otherArgs: [])
             objects += buildMeta.objects
             inputs += buildMeta.inputs
-            linkFlags += buildMeta.linkDependenciesFlags + module.languageLinkArgs
+            linkFlags += buildMeta.linkDependenciesFlags + (try module.pkgConfigArgs().libs)
```

Comment:
I think this should be in the `.Library(.Dynamic)` case, as there is `--static` for static linking once we implement that.